appeals/web/src/server/appeals/question-page/question-page.mapper.js

```diff
@@ -0,0 +1,79 @@
+import { initialiseAndMapAppealData } from '#lib/mappers/appeal.mapper.js';
+import { initialiseAndMapLPAQData } from '#lib/mappers/lpaQuestionnaire.mapper.js';
+
+/**
+ *
+ * @param {string} appealId
+ * @param {string} lpaQId
+ * @returns {{text: string, link: string}}
```

Comment:
As this type is used a couple of times it might be worth creating a named type for it, just a thought